Dockerfile

```diff
@@ -47,14 +43,15 @@ RUN set -ex \
     && pyenv rehash \
     && chmod -R a+w $PYENV_ROOT/shims
 
-RUN pip install --upgrade pip==22.1.2 setuptools==63.1.0 wheel
+RUN pip install --upgrade pip==22.1.2 setuptools==63.1.0 wheel build
 
 RUN pip install \
```

Comment:
Probably should be doing this from a requirements.txt file, or even just from the setup.py requirements. But not a big deal, we're not trying to change this here!